2016-02-04 20:56:35 ERROR [DefaultQuartzScheduler_Worker-8] org.apache.log4j.Logger.error(Logger.java:261) - TableImportJob出现异常：org.springframework.dao.DataAccessResourceFailureException: 
### Error querying database.  Cause: org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
; SQL []; ERROR: canceling statement due to user request; nested exception is org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
ExceptionStackTrace: ---------------start------------------------
	org.springframework.jdbc.support.SQLStateSQLExceptionTranslator.doTranslate(SQLStateSQLExceptionTranslator.java:103)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:72)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	$Proxy9.selectList(Unknown Source)
	org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:198)
	org.bi.manage.app.job.TableImportJob.execute(TableImportJob.java:40)
	org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
ExceptionStackTrace: ---------------end------------------------

2016-02-04 20:56:35 ERROR [DefaultQuartzScheduler_Worker-6] org.apache.log4j.Logger.error(Logger.java:261) - PriodTableImportJob出现异常：org.springframework.dao.DataAccessResourceFailureException: 
### Error querying database.  Cause: org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
### The error may involve AppConfigDataMapper.selectImportTable-Inline
### The error occurred while setting parameters
### Cause: org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
; SQL []; ERROR: canceling statement due to user request; nested exception is org.postgresql.util.PSQLException: ERROR: canceling statement due to user request
ExceptionStackTrace: ---------------start------------------------
	org.springframework.jdbc.support.SQLStateSQLExceptionTranslator.doTranslate(SQLStateSQLExceptionTranslator.java:103)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:72)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:80)
	org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	$Proxy9.selectList(Unknown Source)
	org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:198)
	org.bi.manage.app.job.PriodTableImportJob.execute(PriodTableImportJob.java:41)
	org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
ExceptionStackTrace: ---------------end------------------------

2016-02-04 21:01:30 ERROR [ZkClient-EventThread-25-10.1.235.12:2181] org.I0Itec.zkclient.ZkEventThread.run(ZkEventThread.java:77) - Error handling event ZkEvent[New session event sent to kafka.consumer.ZookeeperConsumerConnector$ZKSessionExpireListener@5b85788e]
kafka.common.ConsumerRebalanceFailedException: GROUPID0_host-10-1-235-12-1454315911318-775c0994 can't rebalance after 4 retries
	at kafka.consumer.ZookeeperConsumerConnector$ZKRebalancerListener.syncedRebalance(ZookeeperConsumerConnector.scala:633)
	at kafka.consumer.ZookeeperConsumerConnector$ZKSessionExpireListener.handleNewSession(ZookeeperConsumerConnector.scala:487)
	at org.I0Itec.zkclient.ZkClient$4.run(ZkClient.java:472)
	at org.I0Itec.zkclient.ZkEventThread.run(ZkEventThread.java:71)
2016-02-04 21:01:30 ERROR [ZkClient-EventThread-42-10.1.235.12:2181] org.I0Itec.zkclient.ZkEventThread.run(ZkEventThread.java:77) - Error handling event ZkEvent[New session event sent to kafka.consumer.ZookeeperConsumerConnector$ZKSessionExpireListener@3b592cb2]
kafka.common.ConsumerRebalanceFailedException: GROUPID0_host-10-1-235-12-1454315912494-d346ffef can't rebalance after 4 retries
	at kafka.consumer.ZookeeperConsumerConnector$ZKRebalancerListener.syncedRebalance(ZookeeperConsumerConnector.scala:633)
	at kafka.consumer.ZookeeperConsumerConnector$ZKSessionExpireListener.handleNewSession(ZookeeperConsumerConnector.scala:487)
	at org.I0Itec.zkclient.ZkClient$4.run(ZkClient.java:472)
	at org.I0Itec.zkclient.ZkEventThread.run(ZkEventThread.java:71)
